core/src/main/java/org/axonframework/eventhandling/scheduling/quartz/QuartzEventScheduler.java

```diff
@@ -198,4 +202,38 @@ public void setTransactionManager(TransactionManager transactionManager) {
     public void setUnitOfWorkFactory(UnitOfWorkFactory unitOfWorkFactory) {
         this.unitOfWorkFactory = unitOfWorkFactory;
     }
+
+    /**
+     * Sets the {@link EventJobDataBinder} instance which reads / writes the event message to publish to the
+     * {@link JobDataMap}. Defaults to {@link }.
```

Comment:
Seems there is a reference to a class name missing in this Javadoc. Assume it needs to be DirectEventJobDataBinder?